core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -78,7 +78,31 @@ angular.module('oppia').directive('adminMiscTab', [
           });
         };
 
-        ctrl.regenerateOpportunitiesRelatedToTopic = function() {
+        ctrl.flushMigrationBotContributionsAsync = async function() {
```

Comment:
Why are these changes part of this PR? I don't expect these changes

Reply:
I have made the changes.